### PR TITLE
Translated into Korean

### PR DESCRIPTION
**Pull request by twoo0220**
Hi, My name is Tae-Woo Lee and I'm a graphics developer from Korea.
I would like to see more Vulkan developer in Korea,
so I would like to ask you to **PR the material translated into Korean.**

### CODE REVIEW

**Comment by twoo0220** [comment]
The extension part is still being translated.

**Comment by spencer-lunarg** [comment]
Hey, thanks for the PR

There is already a `lang/jp` folder for Japanese, I would ask you to move these files from `kor` to `lang/kor` to be consistent

File: README-kor.adoc [hunk 1]
```diff
@@ -0,0 +1,321 @@
+// Copyright 2019-2022 The Khronos Group, Inc.
```
Comment (spencer-lunarg) [inline README-kor.adoc:1]: this file should be in `lang/README-kor.adoc`

Comment (spencer-lunarg) [reply README-kor.adoc:1]: seems there is already one and this is a duplicate

File: README.adoc [hunk 2]
```diff
@@ -1,311 +1,313 @@
...
+:rouge-style: github
+
+image::images/vulkan_logo.png[Vulkan Logo]
+image::images/khronos_logo.png[Khronos logo]
+
+:lang-jp: lang/jp/
+:lang-kor: lang/kor/
+This document is also available in the following languages: +
+xref:{lang-jp}README-jp.adoc[Japanese]
+xref:{lang-kor}README-kor.adoc[Korean]
```
Comment (spencer-lunarg) [inline README.adoc:21]: this should be the only line that is different, not sure why git thinks you changed the entire file

did you edit/save this file with a different encoding?

File: README-kor.adoc (hunk 1 above)
Comment (twoo0220) [reply README-kor.adoc]: I deleted the duplicate files, leaving only the lang/README-kor.adoc file.

File: README.adoc (hunk 2 above)
Comment (twoo0220) [reply README.adoc]: I didn't do any encoding, but I did reset the file and re-commit.
Thanks for the detailed review.

**Review by spencer-lunarg — APPROVED** [review]

**Comment by spencer-lunarg** [comment]
Looks good to me (can't verify the 한국어 is fully accurate) but the files are in their own section.

**Pull request merged**